data/lynis/baseline-lynis-audit-system-nocolors-sle15sp3-x86_64-snapshot7-textmode

```diff
@@ -0,0 +1,630 @@
+
+[ Lynis 2.6.1 ]
```

Comment:
The baseline can be changed regularly, right? do we have some rule to follow which version will be fixed for the baseline in the future?

Reply:
> The baseline can be changed regularly, right? do we have some rule to follow which version will be fixed for the baseline in the future?

The baseline can be changed regularly.
No rules almost, but if we want to change the baseline we need to get the approval from developer.

Reply:
okay, I got it, and so currently we will fix the snapshot7 for baseline in SP3, right?

Reply:
> okay, I got it, and so currently we will fix the snapshot7 for baseline in SP3, right?

yes.